zio-http/src/main/scala/zhttp/service/client/ClientChannelInitializer.scala

```diff
@@ -9,17 +10,27 @@ final case class ClientChannelInitializer[R](
   channelHandler: ChannelHandler,
   scheme: String,
   sslOption: ClientSSLOptions = ClientSSLOptions.DefaultSSL,
+  config: Option[WebSocketClientProtocolConfig] = None,
 ) extends ChannelInitializer[Channel]() {
   override def initChannel(ch: Channel): Unit = {
     val p: ChannelPipeline = ch
       .pipeline()
-      .addLast(new HttpClientCodec)
-      .addLast(new HttpObjectAggregator(Int.MaxValue))
-      .addLast(new ClientResponseHandler())
-      .addLast(channelHandler)
+      .addLast("HTTP_CLIENT_CODEC", new HttpClientCodec)
```

Comment:
Can use variables defined in service/package

Reply:
Yep, these have been moved to the package. This version is just the prototype.